integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,41 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to pull image: %s, %v", out, err)
+	}
+
+	endpoint := "/images/busybox/json"
+	status, body, err := sockRequest("GET", endpoint, nil)
+
+	c.Assert(err, check.IsNil)
+	c.Assert(status, check.Equals, http.StatusOK)
+
+	var inspectJSON types.ImageInspect
```

Comment:
can we call this `imageJSON` instead? `inspectJSON` is a weird name and is kind of similar to a test helper function.